system/global.nix

```diff
@@ -1,4 +1,7 @@
 { pkgs, config, lib, inputs, ... }:
+let
+  asGB = size: toString (size * 1024 * 1024);
```

Comment:
Can this somehow be a percentage of the available disk space? Say, 10%?

Reply:
Seems yes, see this: https://git.dblsaiko.net/systems/tree/common/modules/sys2x/gc.nix

Reply:
Nice catch. I think that was pushed very recently, like.. [10 hours ago](https://www.reddit.com/r/NixOS/comments/13si0vs/check_out_my_nixos_module_to_set_disk_space/).

Reply:
spotted in here: https://old.reddit.com/r/NixOS/

i recommend to subscribe